lib/Traits/Configuration/HelpItemTraits.ts

```diff
@@ -0,0 +1,116 @@
+import objectArrayTrait from "../Decorators/objectArrayTrait";
+import primitiveTrait from "../Decorators/primitiveTrait";
+import ModelTraits from "../ModelTraits";
+
+export enum PaneMode {
+  videoAndContent = "videoAndContent",
+  slider = "slider",
+  trainer = "trainer",
+  externalLink = "externalLink"
+}
+
+export class StepItemTraits extends ModelTraits {
+  @primitiveTrait({
+    type: "string",
+    name: "Title",
+    description: "Title of the step."
+  })
+  title!: string;
+
+  @primitiveTrait({
+    type: "string",
+    name: "Markdown description",
+    description: "Step description in markdown form."
+  })
+  markdownDescription!: string;
+}
+
+export class TrainerItemTraits extends ModelTraits {
+  @primitiveTrait({
+    type: "string",
+    name: "Title",
+    description: "Title of the trainer item."
+  })
+  title!: string;
+
+  @primitiveTrait({
+    type: "string",
+    name: "Footnote",
+    description: "Footnote for the trainer item."
+  })
+  footnote?: string;
+
+  @objectArrayTrait({
+    type: StepItemTraits,
+    name: "Footnote",
+    description: "Footnote for the trainer item.",
+    idProperty: "index"
+  })
+  steps?: StepItemTraits[];
+}
+
+export class HelpItemTraits extends ModelTraits {
```

Comment:
Relating to https://github.com/TerriaJS/terriajs/pull/5918/files#r894124712. If using `itemName` as an id you could add:
```ts
static isRemoval(helpItem: HelpItemTraits) {
  // This condition looks sensible
  return helpItem.markdownText === null && helpItem.trainerItems === null;
}
```
And then you could have default helpItems in terriajs that get overriden/removed in TerriaMap config using the strata system, I think. I have to play around with this branch sometime. And default help items in terriajs means translation support!